frontend/coprs_frontend/commands/add_warning_banner.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from typing import Optional
+
+import click
+from flask import render_template
+
+
+@click.command()
+@click.option("--outage_time", "-o", default=None, help="start time of the outage")
+@click.option("--ticket", "-t", default=None, help="fedora infra ticket ID")
+@click.option("--rest", default=None, help="additional code")
+def add_warning_banner(
+    outage_time: Optional[str], ticket: Optional[int], rest: Optional[str]
+) -> None:
+    """
+    Adds a banner to Copr's frontend page with outage warning. `outage_time` or `rest` is required.
+
+     In case you need to schedule regular outage, fill in only `outage_time` and `ticket`. In case
+    you need to add something else, please use the `rest` parameter to dump to warning banner
+    some additional information. This can be just some text or html code.
+     `rest` parameter can be used as a standalone option just for showing some warning banner
+    with specific information. It can be also piece of HTML code.
+    """
+    if outage_time is None and rest is None:
+        print("Error: `outage_time` or `rest` should be present.", file=sys.stderr)
+        return
+
+    with open("/var/lib/copr/banner-include.html", "w") as banner:
+        banner.write(
+            render_template(
+                "banner-include.html", outage_time=outage_time, ticket=ticket, rest=rest
+            )
+        )
```

Comment:
This would be shorter :-)

```python
text = render_template("banner-include.html", outage_time=outage_time,
                       ticket=ticket, rest=rest)
banner.write(text)
```